Code/GraphMol/Substruct/SubstructMatch.cpp

```diff
@@ -597,16 +616,24 @@ unsigned int RecursiveMatcher(const ROMol &mol, const ROMol &query,
                               SUBQUERY_MAP &subqueryMap,
                               const SubstructMatchParameters &params,
                               std::vector<RecursiveStructureQuery *> &locked) {
+  SubstructMatchParameters lparams = params;
+  // NOTE: maxMatches and recursive queries is problematic. To make this really
+  // cover all cases we'd need a separate parameter for the number of possible
+  // recursive matches. We will add that for the 2022.09 release; for now
```

Comment:
Hasn't the 2022.09 release already passed by?

Reply:
> Hasn't the 2022.09 release already passed by?

ah yeah... I actually mean 2023.03.